common/src/main/res/values/colors.xml

```diff
@@ -27,4 +27,8 @@
     <color name="colorCodeBackground">#f5f5f5</color>
     <color name="colorAlertWarning">#1fffa600</color>
     <color name="colorOnAlertWarning">#ffa600</color>
+    <color name="colorDeviceControlsLightOn">#FDD663</color>
+    <color name="colorDeviceControlsOff">#A6A6A7</color>
```

Comment:
This does not match [the system value](https://android.googlesource.com/platform/frameworks/base/+/766bd4cedbd00fe300f8eb693001395c380fc067/packages/SystemUI/src/com/android/systemui/controls/ui/RenderInfo.kt#87), which is #9AA0A6 instead. I guess this is the first manufacturer difference...

Reply:
those are so close to each other LOL 🤦

Reply:
Then you shouldn't have said you'd matched the system colors 😉